build.sbt

```diff
@@ -2044,7 +2044,8 @@ lazy val examples: ProjectMatrix = (projectMatrix in file("examples"))
       scalaTest.value
     ),
     libraryDependencies ++= loggerDependencies,
-    publishArtifact := false
+    publishArtifact := false,
+    Compile / run / fork := true
```

Comment:
is that temporary for development or a new requirement?

Reply:
It is useful for development, when one wants to run examples with `examples/runMain a.b.c.HelloExample` and terminate a running server.